README.md

```diff
@@ -102,17 +102,16 @@ func main() {
 	// passed in as well to indicate whether the file will be watched and updates
 	// to the file should update the config struct and if the flags name component
 	// separation should use different encoding than lower-kebab-case
-	d, dialsErr := ez.YAMLConfigEnvFlag(ctx, c, ez.WithWatchingConfigFile(true))
+	d, dialsErr := ez.YAMLConfigEnvFlag(ctx, &defCfg, ez.Params[Config]{WatchConfigFile: true})
 	if dialsErr != nil {
 		// error handling
 	}
 
-	// Fill will deep copy the fully-stacked configuration into its argument.
+	// View returns a pointer to the fully stacked configuration file
 	// The stacked configuration is populated from the config file, environment
 	// variables and commandline flags. Can alternatively use
```

Comment:
nit: dangling sentence here...

Reply:
I knew I saw that fragment someplace, but lost track of it 🙁

Removed.